config/frontend.example.toml

```
@@ -90,6 +90,8 @@ runtime_size = 2
 ## Server-side keep-alive time.
 ## Set to 0 (default) to disable.
 keep_alive = "0s"
+# Maximum number of entries in the Prepared Statement cache.
+prepared_stmt_cache_capacity = 10000
```

We use `cache_size` for most cache options, consider using the same style.
```suggestion
prepared_stmt_cache_size = 10000
```